.tekton/pull-request.yaml

```diff
@@ -306,9 +303,9 @@ spec:
                 oc delete --ignore-not-found eventlisteners --all -n $(params.e2e_test_namespace)
       - name: pull-request-status-message
         when:
-          - input: "tasks_pipelines"
+          - input: "execute_e2e"
             operator: "in"
-            values: ["$(tasks.task-switchboard.results.bindings[*])"]
+            values: ["$(tasks.task-switchboard.results.run-e2e)"]
```

Comment:
nitpick: it would make more sense to me to flip the order here

```yaml
input: "$(tasks.task-switchboard.results.run-e2e)"
operator: in
values: ["execute_e2e"]
```

"result in expected values" reads more naturally that "expected value in list of results"

Reply:
Changed now.